### DISCUSSION TITLE
Damaged R-link 2 due to update to higher version

### DISCUSSION THREAD

**Discussion by Kolkol18** [cedricp/ddt4all#1103] [Q&A]
I updated  Renault Koleos 2018 R-Link 2 from v 3.3.16 to V9.xxx and after update camera, media, ac, parking sensors, blind spot and ambient light are not working, please help me to return the r-link 2 to it’s latest proper version.

**Comment by AlMeu** [comment]
@Kolkol18 I think your bricked the unit.
The upgrade from v3.3 to v7.0 is doing in special conditions, usual at dealer service and it is risky anyway.
I can do it also, but I need the unit on my table. 
Or to change the unit directly. 
Anyway if I can help you, I need your email.

Regards

**Reply by eduardoalmosto** [reply]
@Kolkol18 hi, can i need your help to update the R-Link 2 upgrade from v3.3 to v7.0?
